cmd/mr_checkout.go

```diff
@@ -25,10 +25,11 @@ var (
 
 // listCmd represents the list command
 var checkoutCmd = &cobra.Command{
-	Use:   "checkout <id>",
-	Short: "Checkout an open merge request",
-	Long:  ``,
-	Args:  cobra.MinimumNArgs(1),
+	Use:              "checkout <id>",
+	Short:            "Checkout an open merge request",
+	Long:             ``,
+	Args:             cobra.ExactArgs(1),
```

Comment:
This should go back to 
```
cobra.MinimumNArgs(1)
```

Reply:
Oops.  Good catch.  Will fix.

Reply:
Pushed with new version.  Sorry 'bout that.